packages/task/package.json

```diff
@@ -0,0 +1,165 @@
+{
+  "name": "@lit/task",
+  "version": "0.1.0",
```

Comment:
This could be `0.0.0` to make it clear it hasn't been released yet.

Reply:
+1